lldb/source/Plugins/TypeSystem/Swift/SwiftASTContext.cpp

```diff
@@ -3104,6 +3104,10 @@ swift::symbolgraphgen::SymbolGraphOptions &SwiftASTContext::GetSymbolGraphOption
   return GetCompilerInvocation().getSymbolGraphOptions();
 }
 
+swift::CASOptions &SwiftASTContext::GetCASOptions() {
+  return GetCompilerInvocation().getCASOptions();
```

Comment:
Do the CAS options get serialized in the Swift module together with the rest of the invocation?

Reply:
I actually have a follow up patch to include that in the context hash. Does debugger need something from CASOption to be serialized into module?